### ISSUE TITLE
Learning Rate screening: Include Val_loss, 1st derivative,...

### ISSUE THREAD

**Issue by maikherbig** [maikherbig/AIDeveloper#28]
I'd like to inspect not only Loss, but also validation loss after screening learning rates.
Also, often not the absolute loss is of interest, but rather its slope (1st derivative).
A simple drop-down menu which property should be displayed would be nice

**Comment by maikherbig** [comment]
Solved by  
b72d667 
32b67f6 
6b5d36c 
 2ae3a97
